readme.md

```diff
@@ -20,90 +20,92 @@
   </a>
 </p>
 
+[leer en español](readme_es.md)
```

Comment:
Don't you think it would look better this way: 

example: 
[Leer en Español]()

Similarly for the English version.

Reply:
You could also add the link like this

Examples:

```html 
<center><em><a href="readme_es.md">Leer en Español</a></em></center>
```
```html 
<center><em><a href="readme.md">Read this in English</a></em></center>
```

Reply:
ok i will make these changes